.github/workflows/main.yml

```diff
@@ -170,7 +170,7 @@ jobs:
         run: $SBT coverageAggregate
 
       - name: Upload Coverage Report
-        uses: codecov/codecov-action@v3.1.1
+        uses: codecov/codecov-action@3.1.2
```

Comment:
This dropped the `v` in the version number. Looking at the codecov-action tags, there is no "3.1.2" tag, only a "v3.1.2" tag. But it looks like it still worked though. I wonder github actions allows the `v` to be optional? Or maybe codecov-action created the wrong tag, this PR was created, then they fixed it later?

I'll kick off another CI run and see if it still works.

Reply:
Yep, the new CI run failed. I think codecov deleted that wrong tag and added the a new tag. I'll update this PR to use the correct version with a `v` and see if that fixes it.